data/src/main/java/io/gnosis/data/repositories/TokenRepository.kt

```diff
@@ -65,7 +66,7 @@ class TokenRepository(
             TokenType.ETHER,
             ZERO_ADDRESS,
             18,
-            "ETH",
+            BuildConfig.NATIVE_CURRENCY_SYMBOL,
             "Ether",
```

Comment:
Should this and the icon below be adjusted too?

Reply:
Yeah. But to what? :-)

Reply:
xDai I would say

Reply:
Shouldn't you use `NATIVE_CURRENCY_NAME` from the BuildConfig here :D